adr/merk-proofs.md

```diff
@@ -0,0 +1,725 @@
+# Merk Proofs
+
+Merk is a Merkle AVL tree ([https://en.wikipedia.org/wiki/AVL_tree](https://en.wikipedia.org/wiki/AVL_tree))
+
+******************Structure******************
```

Comment:
what this kind of markup stands for? It looks like it should be a level 2 subheading (`## Structure`)

Reply:
fixed